source/data-resources.rst

```diff
@@ -12,14 +12,22 @@ Naive Bayes classifiers trained on:
 
 - `Silva 138 99% OTUs full-length sequences <https://data.qiime2.org/2022.11/common/silva-138-99-nb-classifier.qza>`_ (MD5: ``b8609f23e9b17bd4a1321a8971303310``)
 - `Silva 138 99% OTUs from 515F/806R region of sequences <https://data.qiime2.org/2022.11/common/silva-138-99-515-806-nb-classifier.qza>`_ (MD5: ``e05afad0fe87542704be96ff483824d4``)
-- `Greengenes 13_8 99% OTUs full-length sequences <https://data.qiime2.org/2022.11/common/gg-13-8-99-nb-classifier.qza>`_ (MD5: ``6bbc9b3f2f9b51d663063a7979dd95f1``)
-- `Greengenes 13_8 99% OTUs from 515F/806R region of sequences <https://data.qiime2.org/2022.11/common/gg-13-8-99-515-806-nb-classifier.qza>`_ (MD5: ``9e82e8969303b3a86ac941ceafeeac86``)
```

Comment:
Can you include these links as well so users still have the option to access these? I think that will be helpful for a couple of release cycles since gg 13_8 has been such a staple to the field for so long. For example, I've been using GTDB for a while now, but I almost always classify with both GTDB and GG 13_8, and compare assignments for features of interest. 

If you want to emphasize GG2 over 13_8, you could do something like create a box that says something like:

```
GG2 has succeeded Greengenes 13_8. If you still need to access the outdated 13_8 classifiers, for example to reproduce old results or to compare against new classifiers, you can access them through the following links:
...
```

Reply:
@wasade, your concern makes sense to me. This transition seems to me to be more akin to the QIIME 1 to QIIME 2 transition rather than Silva release transitions (which tend to be relatively frequent incremental updates). Given that this is a major change to a fundamental tool in the field, I think it's important that we prioritize accessibility of both iterations for a release cycle or two. (For the QIIME transition we went through about a one year period where we supported both to allow users to assess which one they preferred and compare results of the two platforms.) 

While the classifiers can be accessed through old versions of the documentation, I think we can make this more accessible. Would it address your concern if we either:
* added a box with a note like I suggested above, but rather than link users to the classifiers directly, point them to [the relevant section of the 2022.11 version of the documentation](https://docs.qiime2.org/2022.11/data-resources/#taxonomy-classifiers-for-use-with-q2-feature-classifier) 
* or bump the old classifier links to the bottom of this page under a heading of something like _Legacy Taxonomic Classifiers_, and note in that section that we recommend using the most recent versions but that these versions may be useful for reproducing old results or comparing to newer classifiers.

Reply:
That makes complete sense, thanks!! I'll issue a commit with the first option